spec.bs

```diff
@@ -16,6 +16,28 @@ Complain About: accidental-2119 yes, missing-example-ids yes
 Indent: 2
 Default Biblio Status: current
 Markup Shorthands: markdown yes
+Assume Explicit For: yes
+</pre>
+
+<pre class="link-defaults">
+spec: html; type: element; text: a
+</pre>
+<pre class="anchors">
+spec: html; type: dfn; urlPrefix: https://html.spec.whatwg.org/multipage/
+  text: serialized state; url: history.html#serialized-state
+  for: URL and history update steps
+    text: serializedData; url: history.html#uhus-serializeddata
+    text: title; url: history.html#uhus-title
+    text: isPush; url: history.html#uhus-ispush
```

Comment:
Semi-related: It seems strange to me that the HTML spec both has _isPush_ and the "history handling behavior". The spec changes proposed here will greatly expand the use of _isPush_. Can/should we deprecate _isPush_ in favor of the "history handling behavior" as a cleanup step?

Reply:
Hmm, I can see both sides. History handling behavior is very tied to the full navigate lifecycle right now (navigate / all the history algorithms). Whereas the URL and history update steps are much smaller scoped. We could reuse history handling (and assert that it's only ever "default" or "replace")... and I guess that's closer to what Chrome does. Hmm.